### PR TITLE
Josayko

### PR DESCRIPTION
**Pull request by josayko**
Just some small adjustemets to the last merge commit: fix login crash because of DTO definition, etc...

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -12,7 +12,7 @@ backend:
 	-cd backend && npm run dev
 
 database:
-	-docker-compose up -d postgres pgadmin
+	-docker-compose up -d postgres
```
Comment (josayko) [inline Makefile:15]: pgadmin has been commented in docker-compose file but we don't need it for development

File: Makefile [hunk 2]
```diff
@@ -40,5 +40,8 @@ ps:
 images:
 	-docker-compose images
 
+exec:
+	-docker exec -it postgres-transcendence bash
```
Comment (josayko) [inline Makefile:44]: because `make exec` is shorter to type :smile:

File: backend/src/api/users/dto/create-user.dto.ts [hunk 3]
```diff
@@ -1,9 +1,23 @@
-import { ApiProperty } from '@nestjs/swagger';
 import { IsDefined, IsNotEmpty, IsOptional, IsString } from 'class-validator';
 
 export class CreateUserDTO {
   @IsString()
   @IsNotEmpty()
   @IsDefined()
   readonly username: string;
+
```
Comment (josayko) [inline backend/src/api/users/dto/create-user.dto.ts:8]: Fix login crash :warning:

File: backend/src/auth/auth.controller.ts [hunk 4]
```diff
@@ -55,5 +55,6 @@ export class AuthController {
   async logout(@Req() req) {
     req.logOut();
     req.session.cookie.maxAge = 0;
+    req.session.destroy();
```
Comment (josayko) [inline backend/src/auth/auth.controller.ts:58]: Wipe out session cookie from `sessions` table in database at logout

File: frontend/src/components/debug/UserDebug.vue [hunk 5]
```diff
@@ -58,7 +58,7 @@ const updateUser = () => {
 
 const deleteUser = (id: number) => {
   Delete(baseUrl + '/' + id.toString()).then(res => {
-    if (res.status == 204) {
+    if (res.status == 200) {
```
Comment (josayko) [inline frontend/src/components/debug/UserDebug.vue:61]: Seems to return `200` instead of `204` when resource is deleted

**Review by josayko — COMMENTED** [review]
Just small fixes => most important is fix login breaks with DTO definition. Nothing special

File: frontend/src/components/debug/UserDebug.vue (hunk 5 above)
Comment (Ourobore) [reply frontend/src/components/debug/UserDebug.vue:61]: Yeah, the API response on DELETE request is 200 OK, as we get what we expect (404 if ressource not found).
To return a 204 No Content, I need to throw an error (what I did before), or maybe @HttpCode() that will be override by the throw 404?

File: backend/src/api/users/dto/create-user.dto.ts (hunk 3 above)
Comment (Ourobore) [reply backend/src/api/users/dto/create-user.dto.ts:8]: Yeah my bad, forgot to add your properties (and it erased the one you defined ^^')

File: Makefile (hunk 1 above)
Comment (Ourobore) [reply Makefile:15]: Sorry, forget to remove pgadmin in Makefile. Have not tested enough...

**Comment by Ourobore** [comment]
Seems good, saw some thing together before. I was scared, though I created more bugs that I didn't see^^

**Comment by Ourobore** [comment]
Can somebody merge and close the request? On smartphone, except closing without committing, I don't seem to be able to merge the pull request?

**Review by AtheDev — APPROVED** [review]

**Pull request merged**